src/Arcus.WebApi.Correlation/CorrelationMiddleware.cs

```diff
@@ -28,7 +28,7 @@ public class CorrelationMiddleware
         /// <exception cref="ArgumentException">When the <paramref name="options"/> doesn't contain a non-<c>null</c> <see cref="IOptions{TOptions}.Value"/></exception>
         public CorrelationMiddleware(
             RequestDelegate next,
-            IOptions<CorrelationOptions> options,
+            IOptions<CorrelationInfoOptions> options,
```

Comment:
So this is a breaking change? 🤔

Reply:
I'd create 2 overloads to and mark this one as obsolete

Reply:
Done.